recipes/default.rb

```diff
@@ -17,3 +17,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+case node['platform_family']
+when "windows"
+	include_recipe 'vcruntime::vc6'
```

Comment:
cookstyle is built into chefdk. If you have ChefDK installed you can just run this from the root of the cookbook.

```
cookstyle -a .
```